src/templates/back/@api/__module_name__/controllers/__bounded_context_name__-find-__module_name__.controller.spec.ts

```diff
@@ -1,19 +1,26 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
-import { Test, TestingModule } from '@nestjs/testing';
+{{
+    setVar 'importsArray' (
+        array
+            (object items=(array 'Testing' 'TestingModule')  path='@nestjs/testing')
+            (object
+                items=
+                (
+                    array
+                    (sumStrings (toPascalCase schema.boundedContextName) 'Find' (toPascalCase schema.moduleName) 'Controller')
+                    (sumStrings (toPascalCase schema.boundedContextName) 'Find' (toPascalCase schema.moduleName) 'Handler')
+                    (sumStrings (toCamelCase schema.boundedContextName) 'Mock' (toPascalCase schema.moduleName) 'Data')
+                )
+                path=(sumStrings config.apiContainer '/' (toKebabCase schema.boundedContextName) '/' (toKebabCase schema.moduleName))
+        )
+    )
+~}}
 {{#if schema.properties.hasI18n}}
-import { CacheModule } from '@nestjs/cache-manager';
+{{ push importsArray
+    (object items=(array 'CacheModule') path='@nest/cache-manager')
+    (object items='commonMockLangData' path=(sumStrings config.appContainer '/common/lang'))
+~}}
```

Comment:
lo mismo, te has saltado las comprobaiones
```suggestion
{{ push importsArray
    (object items=(array 'CacheModule') path='@nest/cache-manager')
~}}
{{#eq schema.boundedContextName 'common'}}
{{ push importsArray
    (object items='commonMockLangData' path=(sumStrings config.appContainer '/common/lang'))
~}}
{{else}}
{{ push importsArray
    (object items='commonMockLangData' path='@aurorajs.dev/common')
~}}
{{/eq}}
```